.devcontainer/devcontainer.json

```diff
@@ -20,8 +20,8 @@
 	},
 	"runArgs": [
 		"--name=${localWorkspaceFolderBasename}_devcontainer",
-		"--cpus=2",
-		"--memory=8gb",
+		// "--cpus=2",
```

Comment:
Should we go ahead and just remove it then?

Reply:
Good point. I've moved it into a doc in our README to make it handy when needed.